virttest/utils_misc.py

```diff
@@ -2557,19 +2557,18 @@ def __get_unit_index(M):
             pass
         return 0
 
-    regex = r"(\d+)\s*(\w?)"
-    match = re.search(regex, value_str)
-    try:
-        value = match.group(1)
-        unit = match.group(2)
-        if not unit:
-            unit = 'B'
-    except TypeError:
+    if value_str.isdigit():
```

Comment:
Check which one, digit or decimal?
the commit message said "Get the whole data when it contains decimals ..."